packages/server/lib/webhook/internal-nango.ts

```diff
@@ -130,6 +138,8 @@ export class InternalNango {
                         });
                     }
 
+                    triggered = true;
+
                     if (webhook === '*') {
                         // Only trigger once since it will match all webhooks
                         break;
```

Comment:
can we not just break in any case? and get rid of triggered

Reply:
The extra break here is handling an edge case if a user for some reason has

```
webhookSubscriptions: ['anEvent', '*']
```

As the code is now it would actually fire twice